roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -95,6 +95,20 @@
     - port: 443
       targetPort: "oaproxy"
 
+- name: Set memcached service
```

Comment:
i would like to avoid adding more to `openshift_logging` as it may eventually be phased out in exchange for a playbook instead